src/tests/roc_audio/test_resampler.cpp

```diff
@@ -101,7 +101,14 @@ TEST(resampler, invalid_scaling) {
     enum { ChMask = 0x1, InvalidScaling = FrameSize };
 
     MockReader reader;
-    ResamplerReader rr(reader, buffer_pool, allocator, config, ChMask, FrameSize);
+    /*
+    core::IAllocator& allocator,
+              const ResamplerConfig& config,
+              packet::channel_mask_t channels,
+              size_t frame_size
+    */
+    BuiltinResampler resampler(allocator, config, ChMask, FrameSize);
+    ResamplerReader rr(reader, resampler, buffer_pool, FrameSize);
```

Comment:
We will need to check all available resampler backends in all tests. We already use this approach in FEC tests:

https://github.com/roc-project/roc/blob/42ed45ed714488f45ff36483e8baaa4f5f3829cc/src/tests/roc_fec/target_openfec/test_writer_reader.cpp#L212-L219

Every test has a for loop that iterates over all FEC schemes. FEC scheme is passed to CodecMap to create appropriate codec.